### PR TITLE
leave stream open in response

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: tools/vcrpy/vcr/stubs/aiohttp_stubs/__init__.py [hunk 1]
```diff
@@ -71,7 +73,7 @@ def build_response(vcr_request, vcr_response, history):
     response._headers = CIMultiDictProxy(CIMultiDict(vcr_response["headers"]))
     response._history = tuple(history)
 
-    response.close()
+    # response.close()
```
Comment (annatisch) [inline tools/vcrpy/vcr/stubs/aiohttp_stubs/__init__.py:76]: Can this just be removed? Or are we still testing?

Comment (xiangyan99) [reply tools/vcrpy/vcr/stubs/aiohttp_stubs/__init__.py:76]: In fact, here we applied a patch to vcr 3.0.0. I keep it just to easier track our change. :)

Comment (xiangyan99) [reply tools/vcrpy/vcr/stubs/aiohttp_stubs/__init__.py]: Removed.

**Review by annatisch — APPROVED** [review]

**Comment by weshaggard** [comment]
@xiangyan99 the path filters end up blocking devops from triggering some of these other pipelines. You can work around this issue by tringgering the non-ci version of the pipeline example `/azp run python - keyvault`

**Pull request merged**